### ISSUE TITLE
'plotman dirs' fails on the command line

### ISSUE THREAD

**Issue by m4tt075** [ericaltendorf/plotman#778]
**Bug Description**

When executed on the CLI, `plotman dirs` fails with the following error message:
```Traceback (most recent call last):
    File "/home/chia-blockchain/venv/bin/plotman", line 8, in <module>
        sys.exit(main())
    File "/home/chia-blockchain/venv/lib/python3.7/site-packages/plotman/plotman.py", line 205, in main
        print(reporting.dirs_report(jobs, cfg.directories, cfg.scheduling, get_term_width()))
    TypeError: dirs_report() missing 1 required positional argument: 'width'      
```

**To Reproduce**

Execute `plotman dirs` on the command line (no interactive mode)

**Expected behavior**

Show directories info, as outlined in `plotman -h`

**System Setup:**

 - OS: OMV5 / Debian 5.10.24~bpo10+1
 - Plotman version: 0.4
 - Method of archiving: None
 
**Config:**

http://paste.debian.net/1200942/

**Additional context & screenshots**

Suspecting a problem with obtaining term information, I've also tried to change to `use_stty_size: False` in the configuration file, but it failed with the same error.

**Comment by altendky** [comment]
Thanks for catching this and my apologies for the hassle.  I've already got one hotfix in for v0.4 and I'll add this.  Might release it later today.  I think it is related to a change in the configuration structure.  Also, right at the moment I have been working on adding type hints and checking for them which would have caught this...  ah well, next time.

**Comment by altendky** [comment]
Alrighty, fixed and committed.  If you wanted to try it out you could use `@release/v0.4.1` instead of `@main` in the installation instructions in the readme.  Thanks again.

**Comment by m4tt075** [comment]
Thank YOU, Sir! And please don't apologize for anything. The tool you've created here to make my and many lifes easier have certainly been a completely different level of hassle...   ;-)   Your efforts are massively appreciated! 

Tried to test as you've advised but files are not found. I think the release tag has not been pushed yet. But please don't worry. I can test when your hotfix is done, too.

**Comment by altendky** [comment]
released in v0.4.1

https://github.com/ericaltendorf/plotman/blob/v0.4.1/CHANGELOG.md

**Comment by m4tt075** [comment]
Many thanks, Eric! Working as expected now. Closing as solved.
